src/rapier_physics.rs

```diff
@@ -46,69 +46,64 @@ pub fn body_description_to_builder(
     for (body_desc, entity, transform) in body_desc_query.iter() {
         commands.entity(entity).remove::<RigidBodyDescription>();
 
-        let isometry = Isometry3::from((transform.translation, transform.rotation));
```

Comment:
We can remove Isometry3 from the import list.

Reply:
This is true. I wasn't really worried about code cleanup, just wanted to get it workin.